### DISCUSSION TITLE
1 modifier (layer) + a sequence of 2 (or more) keys - Possible?

### DISCUSSION THREAD

**Discussion by kenNash** [evan-liu/karabiner.ts#147] [Q&A]
I'm having a lot of fun with this piece of cake, thank you @evan-liu!

I'd like to use my `spacebar` as a layer activator, in which the sequence `ca` would open `Calendar.app`, `ch` `Chrome.app`, and so on.

Would it be possible?

**Comment by evan-liu** [comment]
Do you mean:

1. Press and hold `spacebar`, then press `c`, then press `a`.
2. Press and hold `spacebar`, then press `c` and `a` together.
3. Press `spacebar`, then press `c`, then press `a`.

For `ca`, you can try [mapSimultaneous()](https://karabiner.ts.evanliu.dev/manipulators/from#map-simultaneous).

For `3` (press `spacebar` but without holding it), you can try [Leader Mode](https://karabiner.ts.evanliu.dev/rules/leader-mode).

**Comment by kenNash** [comment]
Thank you @evan-liu.

Both 1 and 3 would be fine for me. I'll have a look at Leader Mode. 🙏

**Reply by evan-liu** [reply]
`1` and `3` are hard to do. Leader mode only works for the first press, but not "press `c` then press `a`" without holding `c`. 

The closest easy way might be [mapSimultaneous()](https://karabiner.ts.evanliu.dev/manipulators/from#map-simultaneous) with `{ key_down_order: 'strict' }` and a long `threshold`, so press-and-hold `c`, then press `a`.

**Comment by kenNash** [comment]
Thank you again, I'll try.
